src/coreclr/nativeaot/System.Private.CoreLib/src/System/InvokeUtils.cs

```diff
@@ -569,18 +569,26 @@ public static object DynamicInvokeParamHelperCore(ref ArgSetupState argSetupStat
 
             Debug.Assert(argSetupState.parameters != null);
             object? incomingParam = argSetupState.parameters[index];
+            bool nullable = type.ToEETypePtr().IsNullable;
```

Comment:
```suggestion
```

Reply:
JFYI: I have moved this from row 583 below to reuse, will move it back

Reply:
Ah ok ... I have not notice that. It was better the way you had it. Could you please revert the suggestion that I have made?

Reply:
Surre